arbitrum-docs/node-running/concepts/arbitrum-nodes-networks.md

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
This is an example of a "stub". Happy to remove if anyone opposes. Basic idea is that we can stub out our known-needed docs and then give our readers a way to signal unmet needs / validate assumptions before doing work.